### ISSUE TITLE
Reporting is broken, crippled MIFs are produced

### ISSUE THREAD

**Issue by Loisel** [pik-piam/remind#14]
`reporting.R` quits early with:
```
[1] "Executing reporting.R"
Error in setNames(pric_emu_base * bio_tax_factor * tdptwyr2dpgj, "Price|Biomass|Bioenergy tax (US$2005/GJ)") :
  dims [product 19] do not match the length of object [0]
> traceback()
6: setNames(pric_emu_base * bio_tax_factor * tdptwyr2dpgj, "Price|Biomass|Bioenergy tax (US$2005/GJ)")
5: setNames(pric_emu_base * bio_tax_factor * tdptwyr2dpgj, "Price|Biomass|Bioenergy tax (US$2005/GJ)")
4: mbind(tmp, setNames(pric_emu_base * bio_tax_factor * tdptwyr2dpgj, 
       "Price|Biomass|Bioenergy tax (US$2005/GJ)")) at reportPrices.R#312
3: reportPrices(gdx, output, regionSubsetList)
2: mbind(output, reportPrices(gdx, output, regionSubsetList)[, t, 
       ]) at convGDX2MIF.R#51
1: convGDX2MIF(infile, file = outfile)
```
on REMIND 779c113d using REMIND library version 36.147.1.
Therefore the MIF file produced only contains a couple of lines.

**Comment by Loisel** [comment]
Related to https://github.com/remindmodel/remind@8225de7f09d2600104a083299b0b8c2141236075

**Comment by dklein-pik** [comment]
Fixed with Version: 36.147.2
